internal/provider/bootstrap_test.go

```diff
@@ -1,54 +0,0 @@
-package provider_test
-
-import (
-	"encoding/json"
-	"io/ioutil"
-	"os"
-
-	"github.com/kairos-io/kairos-sdk/bus"
-
-	. "github.com/kairos-io/provider-kairos/v2/internal/provider"
-	providerConfig "github.com/kairos-io/provider-kairos/v2/internal/provider/config"
-	"github.com/mudler/go-pluggable"
-	. "github.com/onsi/ginkgo/v2"
-	. "github.com/onsi/gomega"
-	"gopkg.in/yaml.v3"
-)
-
-var _ = Describe("Bootstrap provider", func() {
-	Context("logging", func() {
-		e := &pluggable.Event{}
-
-		BeforeEach(func() {
-			e = &pluggable.Event{}
-		})
-
-		It("logs to file", func() {
-			f, err := ioutil.TempFile(os.TempDir(), "tests")
-			Expect(err).ToNot(HaveOccurred())
-			defer os.RemoveAll(f.Name())
-
-			cfg := &providerConfig.Config{
-				P2P: &providerConfig.P2P{
-					NetworkToken: "foo",
-				},
-			}
-			dat, err := yaml.Marshal(cfg)
-			Expect(err).ToNot(HaveOccurred())
-			payload := &bus.BootstrapPayload{Logfile: f.Name(), Config: string(dat)}
-
-			dat, err = json.Marshal(payload)
-			Expect(err).ToNot(HaveOccurred())
-
-			e.Data = string(dat)
-			resp := Bootstrap(e)
```

Comment:
why can't we still have the test?

Reply:
It just tests that it can log to file. IMHO that's a shitty test, it's not testing anything about the functionality of this repo. The logging should be tested on the logger side not here.